### PR TITLE
mail filters

### PR DESCRIPTION
**Pull request by kesselb**
Todo Hamza:

- [x] Use NcSelect with multiple objects for test value selection (wrapped, unlimited width)
- [x] Actions: Add label "Delete action" 
- [x] Divider/Whitespace to group tests and actions
- [x] Delete filter => weird state
- [x] Save filter => success toast
- [x] Form validation => Make it a real form 
- [x] Error state
- [x] MailFilterOperator: Don't modify prop
- [x] Remove operator true
- [x] Enable filter by default
- [x] Change modal title "Name inside modal " to "Update mail filter"
- [x] Check modal size
- [x] Input spacing/width

Todo Daniel:

- [x] Add tests
- [x] Tests with ooo script
- [x] ~Filter definitions~ (follow up)
- [x] To much newlines for the generate script
- [x] Support for quotes in values
- [x] Translation for operators
- [x] Help text

### CODE REVIEW

File: src/components/MailFilters.vue [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+			try {
+				await this.mailFilterStore.update(this.account.id)
+			} catch (e) {
+				// TODO error toast
+			} finally {
+				this.loading = false
+			}
+
+			await this.$store.dispatch('fetchActiveSieveScript', { accountId: this.account.id })
```
[the remark this answers is not in the document]
Comment (kesselb) [reply src/components/MailFilters.vue:163]: Sounds good, do I need to return the new script as result for the update operation?

[the remark this answers is not in the document]
Comment (hamza221) [reply src/components/MailFilters.vue:163]: I'll take care of it in the other PR

**Review by ChristophWurst — APPROVED** [review]
Tested and works :+1: :shipit:

**Review by st3iny — APPROVED** [review]
Ship it! Tested and works.

Eslint is not yet happy though.

**Pull request merged**